front/pages/api/v1/public/files/[token].ts

```diff
@@ -85,9 +85,10 @@ async function handler(
     });
   }
 
+  // TODO(2025-09-22 yuka): remove conversation_participants once migration is done.
   // This endpoint does not support conversation participants sharing. It goes through the private
   // API endpoint instead.
-  if (shareScope === "conversation_participants") {
+  if (shareScope === "conversation_participants" || shareScope === "none") {
```

Comment:
migration newbie here, so I merge this (this won't break anything for existing `conversation_participants` scope (UI will show not published state). Then in a follow up PR I create migration file to move `conversation_participants` to `none`, and clean up the code? 👶